tests/copp/test_copp.py

```diff
@@ -92,6 +93,9 @@ def test_policer(self, protocol, duthosts, enum_rand_one_per_hwsku_frontend_host
             that have a set rate limit.
         """
         duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+        # Skip LLDP/UDLD on Cisco platform(till queue4_group3 issue is fixed)
+        if protocol in ["LLDP", "UDLD"]:
+            skip_release_for_platform(duthost, ["202311"], ["8101", "8102", "8111"])
```

Comment:
8102 seems good, please remove it from the list.

Reply:
Sure @kevinskwang . Do we need this change in master also?